ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -461,7 +461,12 @@ function ClustersTablePanel({
                         </ToolbarGroup>
                     </ToolbarContent>
                 </Toolbar>
-                <Toolbar inset={{ default: 'insetNone' }} className="pf-v5-u-pb-0">
+                <Text className="pf-v5-u-font-size-md">
+                    View the status of services for clusters secured by Advanced Cluster Security
```

Comment:
Maybe work with Kerry and Zhen Peng to adjust wording.
Docs use phrase **secured cluster services** extensively.
docs.redhat.com/en/documentation/red_hat_advanced_cluster_security_for_kubernetes/4.8/html/rhacs_cloud_service/acscs-default-requirements#default-requirements-secured-cluster-services_acscs-default-requirements